rnndb/graph/gf100_pgraph/ctxctl.xml

```diff
@@ -271,7 +287,7 @@
 	<use-group name="gf100_ctxctl"/>
 	<reg32 offset="0x608" name="UNITS">
 		<bitfield low="0" high="4" name="TPC_COUNT"/>
-		<bitfield low="16" high="20" name="UNK_ZCULL_COUNT"/>
+		<bitfield low="16" high="20" name="ZCULL_COUNT"/>
```

Comment:
Do you know what this field actually means?  If not, leave it as UNK.  It's certainly not a "count of ZCULLs", since that doesn't make much sense.  Must be a count of *something* relating to ZCULLs, but AFAIK we have no idea what exactly.